src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -1120,6 +1121,38 @@ because xbuild doesn't support framework reference assemblies.
 
 <!-- Assets Build -->
 
+<Target Name="_CalculateAdditionalResourceCacheDirectories">
+	<CalculateAdditionalResourceCacheDirectories
+			AdditionalAndroidResourcePaths="@(_AdditionalAndroidResourcePaths)"
+			CacheDirectory="$(MonoAndroidIntermediateResourceCache)">
+		<Output TaskParameter="AdditionalResourceCachePaths" ItemName="_AdditonalAndroidResourceCachePaths"/>
+	</CalculateAdditionalResourceCacheDirectories>
+
+	<ItemGroup>
+		<_AdditionalAndroidResourceCachePaths Include="@(_AdditionalAndroidResourcePaths)" />
+	</ItemGroup>
+</Target>
+
+<Target Name="_CreateAdditionalResourceCache"
+		Inputs="@(_AdditionalAndroidResourcePaths->'%(Identity)\AndroidManifest.xml');$(_AndroidBuildPropertiesCache)"
+		Outputs="@(_AdditonalAndroidResourceCachePaths->'%(Identity)\cache.stamp')"
+		DependsOnTargets="_CalculateAdditionalResourceCacheDirectories;_GetAdditionalResourcesFromAssemblies"
+	>
+	<CreateAdditionalLibraryResourceCache
+			AdditionalAndroidResourcePaths="@(_AdditionalAndroidResourcePaths)"
+			AdditionalAndroidResourceCachePaths="@(_AdditonalAndroidResourceCachePaths)">
+		<Output TaskParameter="CopiedResources" ItemName="_AdditionalResourceCache"/>
+	</CreateAdditionalLibraryResourceCache>
+	<Crunch
+			SourceFiles="@(_AdditionalResourceCache)"
+			ToolPath="$(AaptToolPath)"
+			ToolExe="$(AaptToolExe)"
+			YieldDuringToolExecution="$(YieldDuringToolExecution)"
+			Condition=" '$(AndroidExplicitCrunch)' == 'True' "
+	/>
+	<Touch Files="@(_AdditonalAndroidResourceCachePaths->'%(Identity)\cache.stamp')" AlwaysCreate="True" />
+</Target>
+
```

Comment:
I think this came back from a merge or something, these were removed in: https://github.com/xamarin/xamarin-android/commit/1a61fa811e370b68b38a7f318b7d4059dc70779e

Reply:
Removed, thanks for noticing :)